config/environments/production.rb

```diff
@@ -57,7 +57,7 @@
   # config.cache_store = :mem_cache_store
 
   # Use a real queuing backend for Active Job (and separate queues per environment).
-  config.active_job.queue_adapter = :delayed_job
+  config.active_job.queue_adapter = :good_job
```

Comment:
I think we should enforce the `external` mode is always used in production, right? I don't think the async method is meant to be used in prod and we don't want to accidentally run it that way

```ruby
config.good_job.execution_mode = :external
```

Reply:
I think external is the default https://github.com/bensheldon/good_job/blob/51d90b4262070079f34615fcea9e8a8806f7de42/lib/good_job/configuration.rb#L99-L120